client/src/components/ChatRoom.svelte

```diff
@@ -43,8 +49,12 @@
     }
     // create timer
     timer = setInterval(() => {
-      const diff = Math.floor((lobbyData.expiration.toMillis() - Date.now()) / 1000);
-      countdown = diff;
+      if (lobbyData.expiration == undefined) {
+        countdown = GAME_STATE_DURATIONS.CHAT;
```

Comment:
can you initialize countdown with this value instead of waiting for the interval? We wouldn't have to check if it is undefined on `line 97`.

Reply:
Good idea